enterprise/backend/src/metabase_enterprise/embedding/utils.clj

```diff
@@ -0,0 +1,11 @@
+(ns metabase-enterprise.embedding.utils
+  (:require [metabase.models.setting :as setting :refer [defsetting]]
+            [metabase.public-settings :as public-settings]
+            [metabase.public-settings.premium-features :as premium-features]
+            [metabase.util.i18n :refer [deferred-tru]]))
+
+(defsetting pulse-url
+  (deferred-tru "By default \"Site Url\" is used in notification links, but can be overridden.")
+  :getter (fn []
```

Comment:
```suggestion
  (deferred-tru "By default \"Site Url\" is used in notification links, but can be overridden.")
  :visibility :internal
  :getter (fn []
```

We should probably make this `:visibility `:internal` which would mean it's only settable via env var -- since we're not exposing it in the admin UI at this point it makes sense anyway

Reply:
We probably also need to mention this in the documentation somewhere, otherwise nobody will know it exists

Reply:
@camsaul Done